src/execute.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+pipenv install
+pipenv run python main.py '11233, aaabccccdd, aaaaaffffffffffc, abcd, ccceee12eccceee, effeac01cb65c, mattisagoodprogrammer'
```

Comment:
Note the: ⛔️

This is git complaining that there's no empty line at the end of the file. It can create issues when merging. You should be able to configure your IDE to do this automatically.